Detectors/EMCAL/reconstruction/include/EMCALReconstruction/CTFHelper.h

```diff
@@ -129,6 +129,15 @@ class CTFHelper
     value_type operator*() const { return mData[mIndex].getNumberOfObjects(); }
   };
 
+  //_______________________________________________
+  // Trigger word
+  class Iter_trigger : public _Iter<Iter_trigger, TriggerRecord, uint16_t>
+  {
+   public:
+    using _Iter<Iter_trigger, TriggerRecord, uint16_t>::_Iter;
+    value_type operator*() const { return uint16_t(mData[mIndex].getTriggerBits() & 0xffff); }
```

Comment:
@mfasDa @mpoghos I've allocated 16 bits max for the trigger bits. Here I simply take the lowest 16 bits of your 32bit word, this is just a placeholder. Instead, you should build the bit pattern (as short as possible), setting the lowest bits of the returned word. If later you decide to add new bits (still<16 in total) you should add them incrementally w/o touching already defined bits (for backward compatibility).

Reply:
OK, we can simply add a function getTriggerBitsCoded() to o2::emcal::TriggerRecord to encode the trigger bits we are interested in.

Reply:
Yes, will you send me the code to add it to this PR or do this in a separate one? Also, an inverse method should be provided.